ts/components/__tests__/TosWebviewComponent.test.tsx

```diff
@@ -75,157 +75,142 @@ describe("TosWebviewComponent", () => {
       expect(rightButtonHandlerMock).toHaveBeenCalledTimes(1);
     });
   });
-  describe("When rendering and there is an error", () => {
-    it("The error overlay should have been rendered with proper values and the web view should not have been rendered", async () => {
```

Comment:
Even if the new webview lib is missing something, isn't there a way to adjust these tests to check the errors overlay instead of simply disable them all?

Reply:
unfortunately this kind of test was held to test the specific behaviour of the error on the webview to display the error screen, since it is not possible at the moment to manually trigger the onError event of the webview but just test the base component tendering, we cannot reproduce this scenario, any better ideas?